src/utils/validation.js

```diff
@@ -19,6 +18,13 @@ export function email(value) {
   }
 }
 
+const techDomainRegex = /^[A-Z0-9.-]+\.tech$/i
+export function techDomain(value) {
+  if (!isEmpty(value) && !techDomainRegex.test(value)) {
+    return 'Invalid domain'
+  }
+}
```

Comment:
Yesterday we talked about turning this into a generic regex validation, but I think given the current validation setup, that'll actually be quite painful. Let's leave this as is.